data_steward/tools/import_rdr_omop.sh

```diff
@@ -54,7 +54,8 @@ fi
 ROOT_DIR=$(git rev-parse --show-toplevel)
 DATA_STEWARD_DIR="${ROOT_DIR}/data_steward"
 TOOLS_DIR="${DATA_STEWARD_DIR}/tools"
-FIELDS_DIR="${DATA_STEWARD_DIR}/resource_files/fields"
+FIELDS_DIR="${DATA_STEWARD_DIR}/resource_files/schemas"
+RDR_FIELDS_DIR="${FIELDS_DIR}/rdr/"
```

Comment:
There's an extra trailing slash here
```suggestion
RDR_FIELDS_DIR="${FIELDS_DIR}/rdr"
```